src/dbnode/storage/namespace.go

```diff
@@ -791,6 +793,47 @@ func (n *dbNamespace) QueryIDs(
 	return res, err
 }
 
+func (n *dbNamespace) WideQueryIDs(
+	ctx context.Context,
+	query index.Query,
+	opts index.WideQueryOptions,
+) error {
+	ctx, sp, sampled := ctx.StartSampledTraceSpan(tracepoint.NSWideQueryIDs)
+	if sampled {
+		sp.LogFields(
+			opentracinglog.String("query", query.String()),
+			opentracinglog.String("namespace", n.ID().String()),
+			opentracinglog.Int("batchSize", opts.BatchSize),
+			xopentracing.Time("start", opts.StartInclusive),
+			xopentracing.Time("end", opts.EndExclusive),
+		)
+	}
+	defer sp.Finish()
+
+	callStart := n.nowFn()
+	if n.reverseIndex == nil { // only happens if indexing is enabled.
```

Comment:
```suggestion
	if n.reverseIndex == nil { // only happens if indexing is disabled.
```
(but in general the comment seems redundant)

Reply:
Yeah I agree, was mirroring existing comments here, removing them instead.